src/components/SectionHeader/SectionHeader.js

```diff
@@ -0,0 +1,20 @@
+// @flow
+
+import * as React from 'react'
+
+import FloatingShapes from 'views/FloatingShapes'
+import './SectionHeader.scss'
+
+type PropsType = {
+  title: string,
```

Comment:
`background` might be a nice prop to put here to allow the background of the FloatingShapes to change on different pages. You can then implement it via `style={{ background: props.background }}` and use it to pass in colors or images.